Source/WebCore/Modules/mediastream/ImageCapture.h

```diff
@@ -28,6 +28,7 @@
 #if ENABLE(MEDIA_STREAM)
 
 #include "ActiveDOMObject.h"
+#include "Blob.h"
```

Comment:
I am not sure we need ImageCapture to be an ActiveDOMObject, a ContextDestructionObserver might be sufficient, we could look at this in a follow-up